src/tests/ftest/security/cont_create_acl.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. 8F-30005.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from cont_security_test_base import ContSecurityTestBase
+from security_test_base import generate_acl_file
+
+PERMISSIONS = ["r", "w", "rw", "rwd", "rwdt", "rwdtT",
+               "rwdtTa", "rwdtTaA", "rwdtTaAo"]
+
+class CreateContainterACLTest(ContSecurityTestBase):
+    # pylint: disable=too-few-public-methods,too-many-ancestors
+    """Tests container basics including create, destroy, open, query and close.
+
+    :avocado: recursive
+    """
+
+    def test_container_basics(self):
+        """Test basic container create/destroy/open/close/query.
+
+            1. Create a pool (dmg tool) with no acl file passed.
+            2. Create a container (daos tool) with no acl file pased.
```

Comment:
(style)  'pased' may be misspelled - perhaps 'passed'?

Reply:
Done